### ISSUE TITLE
Setup Wizard: Explain users how to start the Setup Wizard again when they attempt to skip it

### ISSUE THREAD

**Issue by alopezari** [woothemes/woocommerce#32195]
### Context
When users skip the Setup Wizard in the first page (Store Details), they can start it later again, but it's not trivial finding out how to do so. We could think about ways to ease that step and improve the user experience.

### Improvement suggestion
When users attempt to skip the wizard, it would be interesting showing them a dialog explaining how they can start the wizard again (or providing them the link to the [documentation](https://woocommerce.com/document/woocommerce-setup-wizard/#reusing-the-setup-wizard)).

### Alternative improvement suggestion
Another approach would be showing in the dialog suggested in the improvement above how to start the wizard again by clicking the "Store Details" task in the Admin/Home page. 

Mentioning both flows in the dialog would be also a valid option.

### Priority
It's not a bug and it's a minor potential improvement of an existing feature, I'd say it's low priority.

### Additional context
In a nutshell, the process of starting again the Setup Wizard is already documented in WooCommerce's site. The improvement request is just about letting users find more easily that piece of documentation.

**Comment by mattsherman** [comment]
@alopezari Thank you for this suggested improvement. I will place it in the product backlog for the development team to consider in the future.

**Comment by alopezari** [comment]
Great, thanks Matt!

**Comment by adrianduffell** [comment]
Closing this out as the setup wizard now makes the store country field required before skipping. I think this should alleviate much of the previous reasons a user might need to go back to the wizard.
